aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,272 @@
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+/>
+
+| Formats                        | Example                                                                                              | When to use                                                                                              | Code snippet (where date is a Javascript Date object)                                                             |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Full written format            | {Intl.DateTimeFormat(undefined, { dateStyle: 'full' }).format(date)}                                 | When space is available and displaying the day of week is beneficial to the user.                        | <Text>`Intl.DateTimeFormat(undefined, { dateStyle: 'full' }).format(date)`</Text>                                 |
+| Long written format            | {Intl.DateTimeFormat(undefined, { dateStyle: 'long' }).format(date)}                                 | When space is available.                                                                                 | <Text>`Intl.DateTimeFormat(undefined, { dateStyle: 'long' }).format(date)`</Text>                                 |
+| Medium written format          | {Intl.DateTimeFormat(undefined, { dateStyle: 'medium' }).format(date)}                               | When space is limited.                                                                                   | <Text>`Intl.DateTimeFormat(undefined, { dateStyle: 'medium' }).format(date)`</Text>                               |
+| Numeric locale-specific format | {Intl.DateTimeFormat(undefined, { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)} | When the user needs to compare across dates, such as in a column or list. Otherwise, use written format. | <Text>`Intl.DateTimeFormat(undefined, { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)`</Text> |
+| ISO 8601 format                | <Text style={{whiteSpace: 'nowrap'}}>{date.toISOString()}</Text>                                     | Log files; trigger times.                                                                                | <Text>`date.toISOString()`</Text>                                                                                 |
+
+Other than log files, date and time should be presented according to the [user's locale](#locale-formatting) and [timezone](#timezones).
+
+For log files, there are two options:
+
+1. Always output timestamps in UTC, indicating UTC in the output.
+1. Allow the user to specify a default timezone and always output timestamps in that timezone, specifying the timezone in the output.
```

Comment:
I'd be careful to include this option for log files. The issue with using a timezone for something that occurred in the past is that it becomes confusing whenever a timezone is changed or if Daylight Savings has occurred. Using UTC in cases such as log files is a much more consistent throughout.

Reply:
I can remove this for now if it's not consistent with your research. I had pulled this from some of Edgar's content and was thinking he must've had some background from products he's worked on.

Reply:
From what I have seen I don't agree with that content but can definitely get his feedback as well. I can share with him the deployment link.

Reply:
I'm fine just removing it for now so we can get this page out. Can always iterate in future.

Reply:
Thinking about this a little bit more and stumbled on this statement:

"- Whenever you are referring to an exact moment in time, persist the time according to a unified standard that is not affected by daylight savings. (GMT and UTC are equivalent with this regard, but it is preferred to use the term UTC. Notice that UTC is also known as Zulu or Z time.)
- If instead you choose to persist a (past) time using a local time value, include the local time offset for this particular time from UTC (this offset may change throughout the year), such that the timestamp can later be interpreted unambiguously."

So yes we can show local time as long as it's offset from UTC (ex. UTC+3). Furthermore, should we clarify in the first bullet point that the timestamp should be displaying UTC+0 specifically? Might also be helpful to include a statement along the lines of "Displaying time zones other than UTC+0 time within log files may present issues if the time zone has changed or if Day Light Savings time has occurred."

Reply:
This topic feels like it needs more discussion. After connecting with Kenny offline, we'll leave this discussion for future. Removing ambiguous wording about log files though.

Reply:
Based on discussion, removed the ambiguous wording but the above comment needs a bit more talking through. Let's address separately.